support-frontend/assets/pages/[countryGroupId]/components/checkoutComponent.tsx

```diff
@@ -709,13 +745,24 @@ export function CheckoutComponent({
 			};
 			setErrorMessage(undefined);
 			setErrorContext(undefined);
-			const createResponse = await fetch('/subscribe/create', {
-				method: 'POST',
-				body: JSON.stringify(createSupportWorkersRequest),
-				headers: {
-					'Content-Type': 'application/json',
+
+			const fetchCreateSupportWorker = () =>
+				fetch('/subscribe/create', {
+					method: 'POST',
+					body: JSON.stringify(createSupportWorkersRequest),
+					headers: {
+						'Content-Type': 'application/json',
+					},
+				});
+
+			const createResponse = await retry(
+				fetchCreateSupportWorker,
```

Comment:
Is creating the support workers request the thing we want to retry? It looks like in the response we get back a `trackingUri` - is that the thing we should be retrying?

Reply:
Crating suppo workers is now not retried.  We retry now in  `processPaymentWithRetries` with eth `trackingUri`